samples/samples_for_tutorials/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "grs-samples",
+  "description": "NodeJS code samples for Google Retails Search API",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "engines": {
+    "node": ">=10"
+  },
+  "files": [
+    "*.js"
+  ],
+  "private": true,
+  "scripts": {
+    "test": "mocha --timeout 600000 test/*.test.js"
+  },
+  "dependencies": {
+    "@google-cloud/bigquery": "^5.9.2",
+    "@google-cloud/retail": "^1.2.2",
+    "@google-cloud/storage": "^5.16.1"
+  },
+  "devDependencies": {
+    "mocha": "^8.0.0",
+    "chai": "^4.3.4",
+    "supertest": "^6.0.0"
```

Comment:
I don't see `supertest` actually being used, can we drop this dependency?

Reply:
removed